### PR TITLE
update to common 9.1.4

### LINKED ISSUE #53: Use strict datatype mapping
Requires updating ex-db-common to https://github.com/keboola/db-extractor-common/releases/tag/9.1.4

### PR DESCRIPTION
**Pull request by pivnicek**
Fixes #53

### CODE REVIEW

File: src/run.php [hunk 1]
```diff
@@ -75,21 +75,21 @@
     $app['logger']->log('info', "Extractor finished successfully.");
     exit(0);
 } catch (UserException $e) {
-    $logger->log('error', $e->getMessage(), $e->getData());
-
+    $logger->log('error', $e->getMessage());
```
Comment (tomasfejfar) [inline src/run.php]: `$logger->error()`?

Comment (tomasfejfar) [reply src/run.php]:
```
} catch (UserException $e) {
    $logger->error($e->getMessage());
    exit(1);
}
```

Comment (pivnicek) [reply src/run.php]: yes, but for synchronous actions we also want to echo the error to stdout I believe.

Comment (tomasfejfar) [reply src/run.php]: There is no difference between the calls IMHO. Just different API to the same method.

Comment (pivnicek) [reply src/run.php]: jj, I know, I was referring to keeping the 
```
if (!$runAction) {
        echo $e->getMessage();
}
```

Comment (tomasfejfar) [reply src/run.php]: oh, my bad! sure!

**Pull request merged**